include/seastar/core/io_queue.hh

```diff
@@ -116,6 +116,9 @@ private:
     size_t _requests_executing = 0;
     uint64_t _requests_dispatched = 0;
     uint64_t _requests_completed = 0;
+    struct flow_monitor;
+    std::unique_ptr<flow_monitor> _flow_mon;
+
```

Comment:
Looks like unique_ptr is unnecessary here, flow_monitor is small and better inlined. I don't completely get why we need it _yet_.

Reply:
I just wanted to keep flow_monitor definition out of .hh file. But I'll move it back, it's small enough indeed